exercises/book-store/description.md

```diff
@@ -36,8 +36,8 @@ This would give a total of:
 
 Resulting in:
 
-- 5 × (100% - 25%) * $8 = 5 × $6.00 = $30.00, plus
-- 3 × (100% - 10%) * $8 = 3 × $7.20 = $21.60
+- 5 × (100% - 25%) \* $8 = 5 × $6.00 = $30.00, plus
```

Comment:
Should those have been changed to `×`?

Reply:
I think so. Probably just over oversight